test/f32-gemm-minmax.cc

```diff
@@ -586,6 +586,275 @@ std::vector<GemmTestParams> CreateTests2(
   return gemm_tests;
 }
 
+std::vector<GemmTestParams> CreateTests3(
+    size_t k_block, size_t adj_k_block,
+    size_t mr, size_t nr, size_t kr, size_t sr,
+    bool is_igemm,
+    std::function<void(GemmMicrokernelTester& tester)> test_func,
+    std::function<void()> isa_check = nullptr) {
+  std::string kbs = std::to_string(k_block);
+  std::string kb2s = std::to_string(k_block * 2);
+  std::string akbs = std::to_string(adj_k_block);
+  nr = nr * xnn_init_hardware_config()->vlenb / sizeof(float);
```

Comment:
`xnn_init_hardware_config()->vlenb` only exists if `XNN_ARCH_RISCV` is `define`d, so I guess this test should be wrapped in an appropriate `#ifdef`?

Reply:
Thank you for catching this. 
I use `postprocess_test_case` to generate marco for `create_test` when `isa == rvv`.